requirements-dev.txt

```diff
@@ -18,7 +18,7 @@ commonmark==0.9.1
 coverage==6.5.0
 craft-archives==0.0.3
 craft-cli==1.2.0
-craft-parts==1.19.3
+craft-parts==1.20.0
```

Comment:
this bump will be removed before merging (it's being addressed in a separate PR). Used this here because I needed a Python plugin fix from that version